xls/visualization/sched_viz/BUILD

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 The XLS Authors
```

Comment:
Neat, should be useful. Feel free to make a separate PR to google/xls with this tool.